lib/l10n.js

```diff
@@ -474,140 +587,198 @@ var l10n = {
+        es: 'Que bueno! Reciba direcciones hacia el lugar donde votar aquí: {url}',
+        ko: "투표소를 가는 길은 여기서 확인 가능합니다: {url}"
     },
     prompt_mail_in: {
-        en: '{{settings.state}} is a vote-by-mail state. Have you mailed in your ballot yet? (yes/no)'
+        en: '{{settings.state}} is a vote-by-mail state. Have you mailed in your ballot yet? (yes/no)',
+        es: '{{settings.state}} es un estado que votar por correo. Usted a enviado su boleto?',
+        ko: "{{settings.state}} 주에서는 우편 투표만 가능합니다. 우편 투표 용지를 보내셨나요? (yes/no)"
     },
     msg_mail_your_ballot: {
         en: 'Please mail your ballot soon - the deadline is coming up! Once you\'ve voted, you can say "I VOTED" and something amazing will happen!',
+        es: 'Por favor envíe su  boleta por correo -- el ultimo dia se acerca! Cuando usted haya votado, puede decir, “YO VOTÉ” y algo increíble pasara!',
+        ko: "투표 용지를 곧 보내셔야 합니다. 마감일이 가깝습니다. 투표한 후 “i voted” 라고 입력하시면 놀라운 일이 일어납니다!"
     },
     msg_share_gotv: {
         en: "Get your friends to the polls -- please share HelloVote!\n* Share on Facebook: {share_url}\n* Share on Twitter: {tweet_url}",
+        es: 'Anime a sus amigos a que vayan a votar -- por favor compartan “HelloVote!\n*” Comparte en Facebook {share_url}\n* Comparte en Twitter  {tweet_url}',
+        ko: "구들에게 알려주세요!\n*페이스북 공유: {share_url}\n*트위터 공유: {tweet_url}"
     },
     msg_share_sms_gotv: {
         en: "Forward this message to everyone you care about! Register to vote and get to the polls: Visit https://hello.vote",
-        es: "¡Reenvíe el mensaje siguiente a sus amigos! Registrese para votar por mensaje de texto: Visite https://hello.vote"
+        es: 'Pase este mensaje a todos los que conozca usted! Regístrate para votar y vaya a votar',
+        ko: "리 알려주세요! 유권자 등록을 하고 투표소 정보를 얻으세요: https://hello.vote"
     },
     msg_i_voted_selfie: {
-        en: "Awesome! Get your 'I VOTED' selfie badge to show your friends & get them to the polls! Visit https://i-voted.hello.vote to get yours."
+        en: "Awesome! Get your 'I VOTED' selfie badge to show your friends & get them to the polls! Visit https://i-voted.hello.vote to get yours.",
+        es: 'Estupendo! Reciba su “I VOTED” para enseñarle a sus amigos. Visita la pagina web https://i-voted.hello.vote para obtener el suyo ',
+        ko: "사합니다! 웹사이트 https://i-voted.hello.vote 에서 I Voted 셀피 뱃지를 받으시고 친구들에게도 투표를 권유하세요! "
     },
     prompt_commit_to_vote_from_notification: {
-        en: "Hey {{first_name}}, it's HelloVote. Election Day is coming up! Would you like to receive a calendar invite with your polling location?"
+        en: "Hey {{first_name}}, it's HelloVote. Election Day is coming up! Would you like to receive a calendar invite with your polling location?",
+        es: 'Hola, {{primer_nombre}} soy “HelloVote” Día de Elección se acerca! Te gustaria recibir una invitación de calendario con la ubicación del lugar a donde votar?',
```

Comment:
The templated variables need to be match the English strings. This should be `{{first_name}}`, not `{{primer_nombre}}`

Reply:
I have fixed all of these